--- a/src/utils.cpp
+++ b/src/utils.cpp
@@ -240,10 +240,10 @@
 
     // restart plasmashell
     QStringList plasmashell = {"plasmashell"};
-    QProcess::execute("/usr/bin/kquitapp6", plasmashell);
+    QProcess::execute("/usr/bin/kquitapp5", plasmashell);
     QProcess::startDetached("/usr/bin/kstart", plasmashell);
   }
 
   // stop krunner, it will be restarted when it is requested again
-  QProcess::startDetached("/usr/bin/kquitapp6", {"krunner"});
+  QProcess::startDetached("/usr/bin/kquitapp5", {"krunner"});
 }
